### PR TITLE
Feature - Add setting for clientIdentificationType

### PR DESCRIPTION
**Pull request by rmi22186**

### CODE REVIEW

File: GoogleAnalyticsEventForwarder.js [hunk 1]
```diff
@@ -521,13 +521,21 @@
...
                     if (forwarderSettings.useLocalhostCookie == 'True') {
-                        ga('create', {
-                            trackingId: forwarderSettings.apiKey,
-                            cookieDomain: 'none',
-                            name: trackerId
-                        });
+                        fieldsObject.cookieDomain = 'none';
+
+                        ga('create', fieldsObject);
+                    } else if (forwarderSettings.useAmpClientId == 'True') {
```
Comment (samdozor) [inline GoogleAnalyticsEventForwarder.js]: Is it legal to set both useLocalhostCookie and useAmpClientId? maybe you could just build up the `fieldsObject` and have a single `ga('create', fieldsObject)` vs the three now?

Comment (rmi22186) [reply GoogleAnalyticsEventForwarder.js]: ah yep

**Comment by rmi22186** [comment]
resolves https://git.corp.mparticle.com/mParticle/issues/issues/4717

**Comment by samdozor** [comment]
@rmi22186 I think this looks good but going to hold off approving until SettingTemplate is pushed. Thanks!

**Pull request merged**